devicetypes/smartthings/zll-rgb-bulb.src/zll-rgb-bulb.groovy

```diff
@@ -110,9 +109,23 @@ def configure() {
 }
 
 def ping() {
+	unschedule()
+	configureHealthCheck()
 	refreshAttributes()
 }
 
+def configureHealthCheck() {
+	Integer hcIntervalMinutes = 12
+	if (!hasConfiguredHealthCheck) {
+		log.debug "Configuring Health Check, Reporting"
+		unschedule("healthPoll")
+		runEvery5Minutes("healthPoll")
```

Comment:
Need to add the `healthPoll` function to this and the other DTHs that don't have it yet